Tests/Test_ReadandSendCarCAN.c

```diff
@@ -0,0 +1,114 @@
+#include "os.h"
+#include "common.h"
+#include "Tasks.h"
+#include "MotorController.h"
+#include "CAN_Queue.h"
+#include "Contactors.h"
+#include "Minions.h"
+
+OS_TCB Task_EnableContactorsTCB;
+CPU_STK EnableContactorsStk[DEFAULT_STACK_SIZE];
+
+
+// set up for the read car can task
+void Task_EnableContactors(void *p_arg) {
+    OS_ERR err;
+
+    // enable systick and the other tasks
+    OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
+
+    CANbus_Init();
+    Contactors_Init();
+    Contactors_Enable(ARRAY_CONTACTOR);
+    Contactors_Enable(ARRAY_PRECHARGE);
+
+    CAN_Queue_Init();
+
+    Minions_Init();
+
+    // delete this task
+    OSTaskDel(&Task_EnableContactorsTCB, &err);
+}
+
+int main(void) {
+    OS_ERR err;
+    OSInit(&err);
+
+    // Initialized with error
+    if (err != OS_ERR_NONE) {
+        printf("OS error code %d\n", err);
```

Comment:
Will this printf work if BSP_UART_Init isn't called?

Reply:
No it won't